client/state/data-layer/wpcom/sites/comments/comment-tree/index.js

```diff
@@ -67,8 +69,62 @@ const announceFailure = ( { dispatch, getState }, { query } ) => {
 	dispatch( errorNotice( error ) );
 };
 
+const fetchCommentsTreeForJetpackSites = ( { dispatch }, action ) => {
+	const { page, siteId, status } = action.query;
+	const query = {
+		fields: 'ID,parent,post,status',
+		number: 100,
+		offset: ( page - 1 ) * 20, // see CommentList COMMENTS_PER_PAGE constant
+		siteId,
+		status,
+		type: 'comment',
+	};
+
+	dispatch(
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ siteId }/comments`,
```

Comment:
@Copons we already have this action under `fetchCommentsList`. Note that we should follow folder conventions noted here: https://github.com/Automattic/wp-calypso/tree/master/client/state/data-layer#file-structure

Since this is an api abstraction that other folks might use, the original name is a better fit.